buildchain/buildchain/image.py

```diff
@@ -80,9 +80,13 @@ def show() -> str:
             path=utils.build_relpath(constants.ISO_IMAGE_ROOT)
         )
 
-    task = targets.Target(task_dep=['_image_pull', '_image_build']).basic_task
+    task = targets.Target(
```

Comment:
minor: code layout could be improved a little bit

Align closing parenthesis:
```
    task = targets.Target(
        task_dep=['check_for:hardlink', '_image_pull', '_image_build']
    ).basic_task
```

Try to keep the command on a single line:

```
task['actions'] = [
    [config.ExtCommand.HARDLINK.value, '-c', constants.ISO_IMAGE_ROOT]
]
```

Reply:
Great, I learnt we have a vertical ruler of 79 enforced. This is the only reason I reformated this. I am going to keep the single line